setup.py

```diff
@@ -9,7 +9,7 @@
     author="Amazon",
     description="Amazon Software License",
     packages=["jupyterlab_pullrequests"],
-    install_requires=["notebook"],
+    install_requires=["notebook", "nbdime", "asynctest"],
```

Comment:
Async test should not be a library dependency. Add it as a "Test" dependency as per https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=tests_require#new-and-changed-setup-keywords